src/commands/url.js

```diff
@@ -30,4 +46,19 @@ export default program => {
     .allowOffline()
     .allowNonInteractive()
     .asyncActionProjectDir(action);
+
+  program
+    .command('url:ipa [project-dir]')
+    .description('Displays the standalone iOS binary URL you can use to download your app binary')
+    .allowOffline()
```

Comment:
These commands probably shouldn't call `allowOffline` (it enables the `--offline` flag), because they require an internet connection and being signed in to Expo to work.